README.md

```diff
@@ -204,7 +204,40 @@ use OpenTok\OpenTok;
 // Force disconnect a client connection
 $opentok->forceDisconnect($sessionId, $connectionId);
 ```
+## Sending Signals
 
+Once a Session is created, you can send signals to everyone in the session or to a specific connection.
+You can send a signal by calling the `signal($sessionId, $options)` method of the
+`OpenTok\OpenTok` class. The `$options` parameter is an associative array used to set the data, type, and connection Id. Please keep in mind
+that the `connectionId` is an optional parameter and should be to used to send a signal to a specific connection.
+
+```php
+use OpenTok\OpenTok;
+
+// Send a signal to a specific connection
+
+$signalOptions = array(
+    'connectionId' => 'da9cb410-e29b-4c2d-ab9e-fe65bf83fcaf',
```

Comment:
Please revise the based on the discussion in https://github.com/opentok/OpenTok-PHP-SDK/issues/208. I think we are deciding on OpenTok.signal($sessionId, $payload, $connectionId).